src/core/lib/http/httpcli_security_connector.cc

```diff
@@ -91,8 +91,17 @@ static void httpcli_ssl_check_peer(grpc_exec_ctx *exec_ctx,
   tsi_peer_destruct(&peer);
 }
 
+static int httpcli_ssl_cmp(grpc_security_connector *sc1,
```

Comment:
Note that in the httpcli case, there are no channel creds.  Should I manually check the parameters that we're feeding into `tsi_create_ssl_client_handshaker_factory()`?

Reply:
We always feed the same params for this specific connector: it's not user controlled so we're good.